usdf/batch.rst

```diff
@@ -321,6 +321,11 @@ to the desired ``site`` subsection.  The ``monitorFilename`` field specifies the
 
   The ``pip uninstall sqlalchemy`` command is needed since the ``pip install 'parsl[monitoring]'`` command installs an earlier version of ``sqlalchemy`` that's incompatible with ``lsst_distrib``.
 
+
```

Comment:
Should the previous example for Parsl monitoring be updated to a more recent weekly?

Also, I believe that parsl is now available in the shared stack at S3DF/USDF in addition to CVMFS.

Reply:
I haven't used that version of the parsl plugin at  USDF in a while, but I can check to see if it still works ok.  if so, i'll update the docs.

Reply:
I slightly reworded so that statement is factually correct without changing the example to a more recent weekly.